### PR TITLE
Update ViewCart - Copy.php

### PR DESCRIPTION
**Pull request by pkManee**
update ViewCart for solve #12

### CODE REVIEW

**Comment by pkManee** [comment]
@Paskorn2  please review now !!!!

**Comment by pkManee** [comment]
hurry !!! @Paskorn2

File: sketch/ViewCart - Copy.php [hunk 1]
```diff
@@ -779,10 +779,15 @@ function getPrice(txt, area) {
 	        });//bootbox
 	    });//fail
 
-	    function viewAllItems() {
-	    	//now implements code to solve issue #10
-	    	//code line 1
-	    	//code line 2
+
+	    function implement_1() {
```
Comment (Paskorn2) [inline sketch/ViewCart - Copy.php:783]: good call

File: sketch/ViewCart - Copy.php [hunk 2]
```diff
@@ -779,10 +779,15 @@ function getPrice(txt, area) {
...
-	    	//now implements code to solve issue #10
-	    	//code line 1
-	    	//code line 2
+
+	    function implement_1() {
+	    	//now code to solve issue #12
+	    	//code line ....
+	    }
+
+	    function implement_2() {
```
Comment (Paskorn2) [inline sketch/ViewCart - Copy.php:788]: good call too

**Review by Paskorn2 — APPROVED** [review]

**Comment by pkManee** [comment]
close #12

**Pull request merged**